Libraries/Lists/MetroListView.js

```diff
@@ -155,6 +157,11 @@ class MetroListView extends React.Component<Props, $FlowFixMeState> {
         ref={this._captureRef}
         renderRow={this._renderRow}
         renderFooter={this.props.FooterComponent && this._renderFooter}
+        renderHeader={
+          this.state.ds.getRowCount()
```

Comment:
Looking at the VirtualizedList implementation I think we still need to render the header if the list is empty. Have you tested header + empty component to make sure the behaviour is the same?

Reply:
Thanks for the review, and sorry for the delay, I'll check this hopefully tomorrow night.